doc/building-and-deploying-the-website.md

```diff
@@ -0,0 +1,20 @@
+## Building and Deploying (Automated)
+
+### Testing
+
+After you are finished developing and making your changes, make sure to test them. Run:
+
+	npm test
+
+`npm test` runs both [eslint] and [mocha] tests. If your tests pass, commit and push your work to Github.
+
+### Travis 
+
+[Travis] automatically builds and publishes the website on every change. In [travis.yml](../.travis.yml), Travis  installs required dependencies and runs the build script. Travis will build the full website for you by running `gulp build --prod` under the hood. Travis also uses [SVN] to update, copy, add, and commit the new changes over to the website. Committing to svn can only occur once the commit has been merged to master. You can read more about is happening under the hood with SVN [here](deploying-the-website.md). Travis also runs `npm test` and will notify you if any of your `eslint` or `mocha` tests are failing. When Travis is done building and deploying, send a pull request and ask for a review.
```

Comment:
> When Travis is done building and deploying, send a pull request and ask for a review.

Reply:
Ok, thought about that a bit.

Reply:
yes. Tests will run on branches/prs. Deploy only happens on master

Reply:
Ok, maybe this should be explain a bit here and the workflow clarified?

You fork, commit your changes to your fork, wait for the test result via Travis.
If green, you can send the PR which will trigger Travis again.
If (green and) merged, it will trigger a deploy.

Correct?

Reply:
@stevengill what ensures that deploy happens only on master?

Reply:
@stevengill What about forks in the apache repo? I did create one of those by accident instead of forking on my user for #761 - there probably has to be a real check for `master`?